yt/frontends/nifti/data_structures.py

```diff
@@ -9,7 +9,7 @@
 from yt.funcs import setdefaultattr
 from yt.geometry.grid_geometry_handler import GridIndex
 from yt.utilities.on_demand_imports import NotAModule, _nibabel as nib
-
+from yt.geometry.geometry_enum import Geometry
```

Comment:
to align with the how it's done everywhere else in the code base
```suggestion
from yt.geometry.api import Geometry
```